README.md

```diff
@@ -7,43 +7,45 @@ cmsrel CMSSW_10_6_8_patch1
 cd CMSSW_10_6_8_patch1/src
```

Comment:
The recommended CMSSW version for UL is CMSSW_10_6_12

```suggestion
cmsrel CMSSW_10_6_12
cd CMSSW_10_6_12/src
```

Reply:
@smoortga Done